docs/docsite/Makefile

```diff
@@ -35,7 +35,7 @@ docs: clean htmldocs
 generate_rst: staticmin
 
 htmldocs: generate_rst
-	cd ../../lib/ansiblelint; python ./generate_docs.py
+	python -m ansiblelint.generate_docs
```

Comment:
I'd rather stick it directly into tox.ini. But this could be done in a separate PR